docs/user/troubleshooting.md

```diff
@@ -52,7 +52,9 @@ If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
 ### Unable to SSH into Master Nodes
 
-In order to SSH into the master nodes, it is necessary to include an administrator's SSH key during the installation. If SSH authentication is failing, ensure that the proper SSH key is being used.
+In order to SSH into the master nodes as user `core`, it is necessary to include an administrator's SSH key during the installation. When asked by the installation wizzard for the ssh-key, make sure you define the absolute path to your key. This key will be added to the core user's authorized_keys file. If installed in aws you will not see this key attached to your instance as a keypair in the aws console. 
```

Comment:
nits: "ssh" -> "SSH".  "SSH" is the protocol; `ssh` is just the executable.  And "core" -> "`core`" and "authorized_keys" -> "`authorized_keys`".  And "aws" -> "AWS".

Re: "absolute path", [the wizard prompt is a `Select`][1] of your `~/.ssh/*.pub`.  I don't think it allows free-form input (or at least, it didn't allow it in my tests).

[1]: https://github.com/openshift/installer/blob/b4f5ceb6bfde8d3dc0e29f708e0494488ea37ee0/pkg/asset/installconfig/ssh.go#L99

Reply:
minor: "wizzard" -> "wizard"